### ISSUE TITLE
Beginner Question - Focus on TextArea

### ISSUE THREAD

**Issue by simmihugs** [fjvallarino/monomer#80]
TextArea does not allow me to type in it, unless I click with my Mouse into the Textarea.

My application has a TextField. When I set the focus to the TextField with `SetFocusOnKey "textField1"` 
--> I can type in the TextField right away; As expected

My app also has a TextArea. When I set the focus to the TextArea with `SetFocusOnKey "textArea1"`
--> I cannot type in the TextArea unless I click with my Mouse into the TextArea
The TextArea, I did not change the the theme, has a blue border when it is on Focus, but there is no Cursor
--> blue border so it is focused 

How do I make TextAreas accept input after receiving focus?

P.S. if there is a better way to ask trivial questions like this, let me know

**Comment by fjvallarino** [comment]
Hi! This is indeed a library issue. The problem is `textArea` is automatically wrapped by a `scroll` widget, so the `nodeKey` call assigns the key to `scroll` and not to `textArea`. Automatically adding the scroll causes some complications (such as this), but I think it's generally the expected behavior (otherwise, users would need to wrap it manually).

I just pushed a new branch with a fix. You can test it by modifying your `stack.yaml` and adding these lines (commenting out the line with `- monomer-1.3.0.0`).

```yaml
- git: https://github.com/fjvallarino/monomer.git
  commit: 68acc4c917f512eae143532388f80884bebe4b07
```

It's totally fine to ask usage questions here!

**Comment by simmihugs** [comment]
Hi @fjvallarino,
now it works as I expected, great thanks.

**Comment by fjvallarino** [comment]
Sure, you can close it if you feel the issue is fixed!

By the way, I just merged the PR to `main` and deleted the associated branch. Because of this, the commit hash you tested with does not exist anymore (if you try a clean build, it will fail because `stack` will not find the hash). The new hash from the `main` branch is `510de85d5c48d16d45464af96035b0221aca7b09` (this commit hash won't change/be removed).

This fix will be included whenever 1.4 is released, and I also hope to include the fix to the scroll thumb-size issue.

Thanks!

**Comment by simmihugs** [comment]
Sounds good - looking forward to the 1.4 release
